### PR TITLE
Improve `sql.crud`

### PR DESCRIPTION
**Pull request by bryanforbes**
### Description
Improve `sql.crud`. Depends on #26.

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [x] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: sqlalchemy-stubs/sql/crud.pyi [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+from . import type_api
+from ..util import langhelpers
 
-REQUIRED: Any
+_TE = TypeVar("_TE", bound=type_api.TypeEngine[Any])
 
-class _multiparam_column(elements.ColumnElement):
+REQUIRED: langhelpers._symbol
+
+class _multiparam_column(elements.ColumnElement[_TE]):
```
Comment (CaselIT) [inline sqlalchemy-stubs/sql/crud.pyi:12]: isn't this class private?

Comment (bryanforbes) [reply sqlalchemy-stubs/sql/crud.pyi:12]: It seems to be. When the stubs were generated it was retained, so I filled it out.

Comment (CaselIT) [reply sqlalchemy-stubs/sql/crud.pyi:12]: maybe it's used somewhere else, so it was retained because if it?

Comment (bryanforbes) [reply sqlalchemy-stubs/sql/crud.pyi:12]: That's what I was thinking, but it doesn't seem to be used anywhere

Comment (CaselIT) [reply sqlalchemy-stubs/sql/crud.pyi:12]: @zzzeek ideas?

Comment (zzzeek) [reply sqlalchemy-stubs/sql/crud.pyi:12]: it's used here:  https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/sql/crud.py#L812

would be used in a statment like this:

```python
>>> from sqlalchemy import table, column
>>> t = table('t', column('a'), column('b'))
>>> stmt = t.insert().values([{"a": 1}, {"a": 2}])

```

looks like the default stringify needs to be opened up for that...

Comment (CaselIT) [reply sqlalchemy-stubs/sql/crud.pyi:12]: I guess we can decide later if we want this or not

**Pull request merged**